app/blueprints/fund_builder/routes.py

```diff
@@ -254,42 +254,50 @@ def fund(fund_id=None):
+                fund.name_json["cy"] = form.name_cy.data
+                fund.title_json["en"] = form.title_en.data
+                fund.title_json["cy"] = form.title_cy.data
+                fund.description_json["en"] = form.description_en.data
+                fund.description_json["cy"] = form.description_cy.data
+                fund.welsh_available = form.welsh_available.data == "true"
+                fund.short_name = form.short_name.data
+                fund.audit_info = {"user": "dummy_user", "timestamp": datetime.now().isoformat(), "action": "update"}
+                fund.funding_type = form.funding_type.data
+                fund.ggis_scheme_reference_number = (
+                    form.ggis_scheme_reference_number.data if form.ggis_scheme_reference_number.data else ""
+                )
+                update_fund(fund)
+                flash(f"Updated fund {form.title_en.data}")
+                return redirect(url_for("build_fund_bp.view_fund", fund_id=fund.fund_id))
+
+            new_fund = Fund(
+                name_json={"en": form.name_en.data},
+                title_json={"en": form.title_en.data},
+                description_json={"en": form.description_en.data},
+                welsh_available=form.welsh_available.data == "true",
+                short_name=form.short_name.data,
+                audit_info={"user": "dummy_user", "timestamp": datetime.now().isoformat(), "action": "create"},
+                funding_type=FundingType(form.funding_type.data),
+                ggis_scheme_reference_number=(
+                    form.ggis_scheme_reference_number.data if form.ggis_scheme_reference_number.data else ""
+                ),
             )
-            update_fund(fund)
-            flash(f"Updated fund {form.title_en.data}")
-            return redirect(url_for("build_fund_bp.view_fund", fund_id=fund.fund_id))
-
-        new_fund = Fund(
-            name_json={"en": form.name_en.data},
-            title_json={"en": form.title_en.data},
-            description_json={"en": form.description_en.data},
-            welsh_available=form.welsh_available.data == "true",
-            short_name=form.short_name.data,
-            audit_info={"user": "dummy_user", "timestamp": datetime.now().isoformat(), "action": "create"},
-            funding_type=FundingType(form.funding_type.data),
-            ggis_scheme_reference_number=(
-                form.ggis_scheme_reference_number.data if form.ggis_scheme_reference_number.data else ""
-            ),
-        )
-        add_fund(new_fund)
-        flash(f"Created fund {form.name_en.data}")
-        return redirect(url_for(BUILD_FUND_BP_INDEX))
+            add_fund(new_fund)
+            flash(f"Created fund {form.name_en.data}")
+            return redirect(url_for(BUILD_FUND_BP_INDEX))
 
-    error = error_formatter(form.errors)
+        error = {**error,
+                 'short_name': [f'Given fund short name already exists.']}
```

Comment:
this doesn't seem familiar to me as GOV.UK error message content style - do we want to run this through content design? Or is that all happening separately?

Reply:
We have a ticket to verify the error message with content designer @chaneldiep